server/api/map_handler.py

```diff
@@ -0,0 +1,71 @@
+
+import json
+import smtplib
+import requests
+from flask import jsonify, request, Blueprint
+from config import API_KEY
+map_handler = Blueprint('map_handler', __name__)
+
+@map_handler.route('/map', methods=['POST'])
+def map():
+    if request.method != 'POST':
+        return 'NOT POST'
+    
+    postData = json.loads(request.get_data())
+    address = postData['address'].replace(' ', '+')
+    
+    url = "https://maps.googleapis.com/maps/api/geocode/json?address=%s&key=%s" % (address, API_KEY)
+    googleReq = requests.get(url).json()
+
+    #check for bad request
+    #check if location exists
+    googleHandler = googleAPIHandler()
+    status = googleHandler.handle(googleReq)
+    if status != True:
```

Comment:
```suggestion
    if not status:
```

Reply:
Or `if status is not True:`.
Both are the more "pythonic" way of doing this check

Reply:
will if not status be true if it is not undefined or null variable

Reply:
I'll just change it to if status != 'OK' in order to prevent confusion